backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5381,3 +5381,21 @@ databaseChangeLog:
             REVOKE SELECT ON TABLE ${database.defaultSchemaName}.role FROM ${noPhiUsername};
             REVOKE SELECT ON TABLE ${database.defaultSchemaName}.api_user_facility FROM ${noPhiUsername};
             REVOKE SELECT ON TABLE ${database.defaultSchemaName}.api_user_role FROM ${noPhiUsername};
+  - changeSet:
+      id: add-flu-a-and-b-to-supported_disease-table
+      author: tmz1@cdc.gov
+      comment: Add Flu A and B to the supported_disease table.
+      changes:
+        - tagDatabase:
+            tag: add-flu-a-and-b-to-supported_disease-table
+        - sql:
+            sql: |
+              INSERT INTO ${database.defaultSchemaName}.supported_disease (
+                   internal_id,
+                   name,
+                   loinc)
+              VALUES
+                   (gen_random_uuid(), 'Flu A and B', '44567-6')
```

Comment:
Yeah I think I have a story somewhere to deprecate this column altogether - I know we've talked about how the values in here are inconsistent disease-to-disease and we don't actually use them for anything.  If someone argued passionately for making it `FooBarBaz` I would say 🤷 sure why not.

Reply:
I will argue but not very passionately for making it LP14402-9 since that's actually referring to the disease, whereas 44567-6 is specifically for Flu A/B in the nose, which isn't guaranteed correct since that will depend on the device

I figure while the data exists in prod, we should at least try to make it correct in case it does end up getting used at some point

Reply:
imo that's a nit though so I hit approve